cmd/introspect_cfg.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+
+	oidc "github.com/jentz/vigilant-dollop"
+)
+
+func parseIntrospectFlags(name string, args []string) (runner CommandRunner, output string, err error) {
+	flags := flag.NewFlagSet(name, flag.ContinueOnError)
+	var buf bytes.Buffer
+	flags.SetOutput(&buf)
+
+	var oidcConf oidc.Config
+	flags.StringVar(&oidcConf.IssuerUrl, "issuer", "", "set issuer url (required)")
+	flags.StringVar(&oidcConf.DiscoveryEndpoint, "discovery-url", "", "override discovery url")
+	flags.StringVar(&oidcConf.IntrospectionEndpoint, "token-url", "", "override token url")
```

Comment:
since this is introspection I would have thought the argument would be introspection-url

Reply:
100% - corrected.